bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,9 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if not all(revision.contains(issue) for issue in issues if issue.is_publishable):
+            comment = comment.replace(":", ": (defects might be in the parent stack)")
```

Comment:
Please don't use replace, it makes no sense.

```suggestion
            comment += ": (defects might be in the parent stack)"
```